ams/server/src/main/java/com/netease/arctic/server/table/executor/OrphanFilesCleaningExecutor.java

```diff
@@ -325,9 +325,12 @@ private static Set<String> getValidMetadataFiles(UnkeyedTable internalTable) {
           cnt,
           size);
     }
-    Stream.concat(
+    Stream.of(
             ReachableFileUtil.metadataFileLocations(internalTable, false).stream(),
```

Comment:
Amoro project uses 2 spaces as indent, can you fix the format here?

Reply:
OK, I will check it.